warehouse/query-core/src/main/java/datawave/query/jexl/visitors/PushdownLargeFieldedListsVisitor.java

```diff
@@ -127,14 +127,14 @@ public Object visit(ASTOrNode node, Object data) {
                 List<JexlNode> markers = new ArrayList<>();
                 
                 try {
-                    // if we have an hdfs cache directory and if past the fst threshold, then create the fst and replace the list with an assignment
-                    if (fstHdfsUri != null && (eqNodes.size() >= config.getMaxOrExpansionFstThreshold())) {
+                    // if we have an hdfs cache directory and if past the fst/list threshold, then create the fst/list and replace the list with an assignment
+                    if (eqNodes.size() >= config.getMaxOrExpansionThreshold()) {
+                        markers.add(ExceededOrThresholdMarkerJexlNode.createFromValues(field, values));
+                        eqNodes = null;
+                    } else if (fstHdfsUri != null && (eqNodes.size() >= config.getMaxOrExpansionFstThreshold())) {
```

Comment:
I think the fst check needs to be first the way we were doing before.  If we are favoring the fst too much then the fst threshold needs to be set larger.

Reply:
i suppose you're right.  fixed.